src/store/flow/block.ts

```diff
@@ -73,6 +73,17 @@ export const mutations: MutationTree<IFlowsState> = {
 }
 
 export const actions: ActionTree<IFlowsState, IRootState> = {
+  block_setLabel({commit}, {blockId, value}) {
+    const snakeCase = (value: string) => value?.trim()
+      .replace(/[^\w\s]+/g, '')
+      .split(' ')
+      .map((word) => word.toLowerCase())
+      .join('_')
```

Comment:
Curiosity Q: Does [lodash's snakeCase](https://lodash.com/docs/4.17.15#snakeCase) behave similar enough that we could use it here instead of hand-rolling?

Reply:
Yes, that would work!  I did not realize lodash had a function for it, thanks for sharing 👍